Makefile

```diff
@@ -85,13 +85,9 @@ ifeq ($(SUBMODULES), true)
 endif
 	./hack/prepare-venv.sh
 
-.PHONY: ipu_host
-ipu_host: prepare-e2e-test
-	./hack/ipu_host_deploy.sh
-
-.PHONY: ipu_deploy
-ipu_deploy: prepare-e2e-test
-	./hack/ipu_deploy.sh
+.PHONY: deploy_clusters
+deploy_clusters: prepare-e2e-test
+	bash hack/both.sh
```

Comment:
why adding all those make file targets? The script can be called directly. If it has a good name it's also discoverable.